### PR TITLE
Adding Emoji Rate and the logic for the Settings-options forms

### PR DESCRIPTION
**Pull request by Hai5edfm**
This PR adds the logic for the #54 , also includes the #66  EmojiRate logic and css, implemented in the FilmView (will be implemented in the home-cards once the DB is able to)

image
image
image

### CODE REVIEW

File: src/components/FilmCard/index.js
Comment (Hai5edfm) [reply src/components/FilmCard/index.js:38]: yeah this is the Ivan's idea about showing the emoji rate in the Home-film-card, it wouldn't have the functionality of "change it" and it abviously would be an "add-reaction" icon until the user marks that movie as watched and then sends a rate :v, to sum it up, it's just visual, the real function is in the film-view

File: src/components/EmojisRate/index.js
Comment (Hai5edfm) [reply src/components/EmojisRate/index.js:3]: I think both require almost same resources to be charged, and the real difference is in the use, in this case I think we should add the icons in SVG(because they are not images, they are buttons) but I added the .png because they were more "easy" to implement and in this case we do not even have a real deployment with storage to have them :v, in the future I could try to see how to modify the SVG's colors (wrote down)

File: src/components/FilmCard/FilmCard.scss
Comment (Hai5edfm) [reply src/components/FilmCard/FilmCard.scss:90]: okay, I just wanted to make the font-size bigger :v, but yeah, you are right, I should change the size in the variables of that :v, Idk why that same tag is duplicated in the same css doc xd, but yeah xd, we need to take a look of that sort of things

Comment (LilianaRestrepoTorres) [reply src/components/FilmCard/FilmCard.scss:90]: Exactly. If you don't mind and it's possible you can add that detail in this PR, before Ivan approve it. And so, we won't have this pending.

**Comment by LilianaRestrepoTorres** [comment]
That's great. Only requires the Ivan's approval.

File: src/components/FilmCard/FilmCard.scss
Comment (Hai5edfm) [reply src/components/FilmCard/FilmCard.scss]: done

File: src/components/EmojisRate/index.js
Comment (AreYouIvan) [reply src/components/EmojisRate/index.js:3]: **SVG**: These are vectors and it uses an algorithm to reorganized their pixels when their size is changed, these are the best choice to use in icons and some kinds of logos.
**PNG**: These are images that can handle 16 million colors, these are the best choice to use in images with high quality, this format is much larger than SVG.
To handle icons will be always better to use SVG, however, I don't know if using some icons with PNG format could affect our small application, but the truth is that the best practice would be using icons with SVG format.

[Difference between these formats by Adobe
](https://www.adobe.com/creativecloud/file-types/image/comparison/png-vs-svg.html#:~:text=While%20PNGs%20are%20capable%20of,size%20without%20losing%20their%20resolution.)

File: src/components/profileConf/profileConfig/index.js
Comment (AreYouIvan) [inline src/components/profileConf/profileConfig/index.js:2]: The code seems well, but the nomenclature to declare components in React is **PascalCase**, so the right way to name the components would be: ChangingUser (Let's try to be the more descriptive as we can), ChangingPass, ProfileConf, ProfileConfig just like the function components.

Also, I saw there is a component inside another component in the folders just like this: _profileConf/profileConfig/index.js, changingPass, changingUsr_. 

Every component should be separated, or if ChangingUser and ChangingPass are components that only will be rendered in a specific part of the application you can use the folder "Containers" to store them just like this: _/containers/ProfileConfig/index.js_ <- over here you would import the components, this way the project is more readable when we need to debug or refactorize.

Besides, I didn't view the functionality of the first folder "profileConf" which is wrapping the "profileConfig" folder.

I won't request changes because the imports will be affected, but I advise using the current file composition because it's cleaner. 😉 

This is the doc about the recommendations to make easier the development with React 😄 
https://es.reactjs.org/docs/faq-structure.html#grouping-by-features-or-routes

File: src/components/profileConf/profileConfig/index.js
Comment (AreYouIvan) [inline src/components/profileConf/profileConfig/index.js:5]: Also, if the folder has the same name as the react component "ChanginPass" (There's a typo) you can just use in the import the following: 
`import { ChangingPass } from "./ChangingPass"` without the index.js because with babel it's implicit

**Review by AreYouIvan — APPROVED** [review]

File: src/components/EmojisRate/index.js
Comment (LilianaRestrepoTorres) [reply src/components/EmojisRate/index.js:3]: @Hai5edfm @AreYouIvan Thank you guys. My question was more related with when is better include either images (SVG, PNG that you explained amazing) or import icons from an external source? I know that it won't have an important impact in our app, it's only to know how that works in bigger projects.

Comment (AreYouIvan) [reply src/components/EmojisRate/index.js:3]: Import the icons from an external source absolutely, the best practice would be to avoid sending binaries as icons/images to our repo. 😄

**Pull request merged**
